pyproject.toml

```diff
@@ -62,6 +62,7 @@ sqlalchemy = "*"
 tortoise-orm = "*"
 tox = "*"
 uvicorn = "*"
+cryptography = "*"
```

Comment:
Should we treat this dependency the same way we do `brotli`, i.e., optional and include in full extras?

Reply:
We could, but what s the advantage of doing this?

Reply:
Size of venv with only starlite is 92M, size of venv with starlite and cryptography is 108MB.. if it is only needed for session middleware why should it be non-optional?

Reply:
It is optional - these are dev dependencies

Reply:
oh ha, right - it isn't named in the prod dependencies section though.